include/deal.II/numerics/vector_tools_boundary.h

```diff
@@ -206,53 +224,34 @@ namespace VectorTools
+    const types::boundary_id                   boundary_indicator,
     const Function<spacedim, number> &         boundary_function,
     std::map<types::global_dof_index, number> &boundary_values,
     const ComponentMask &component_mask = ComponentMask());
 
 
-  /**
-   * Call the other interpolate_boundary_values() function, see above, with
-   * <tt>mapping=MappingQ@<dim,spacedim@>(1)</tt>. The same comments
-   * apply as for the previous function, in particular about the use of the
-   * component mask and the requires size of the function object.
-   */
-  template <int dim, int spacedim, typename number>
-  void
-  interpolate_boundary_values(
-    const DoFHandler<dim, spacedim> &dof,
-    const std::map<types::boundary_id, const Function<spacedim, number> *>
-      &                                        function_map,
-    std::map<types::global_dof_index, number> &boundary_values,
-    const ComponentMask &component_mask = ComponentMask());
-
-
   /**
    * Insert the (algebraic) constraints due to Dirichlet boundary conditions
-   * into a AffineConstraints @p constraints. This function identifies the
+   * into an AffineConstraints object. This function identifies the
    * degrees of freedom subject to Dirichlet boundary conditions, adds them to
    * the list of constrained DoFs in @p constraints and sets the respective
    * inhomogeneity to the value interpolated around the boundary. If this
    * routine encounters a DoF that already is constrained (for instance by a
    * hanging node constraint, see below, or any other type of constraint, e.g.
-   * from periodic boundary conditions), the old setting of the constraint
-   * (dofs the entry is constrained to, inhomogeneities) is kept and nothing
-   * happens.
+   * from periodic boundary conditions), the old setting of the constraint is
+   * kept and nothing happens.
    *
    * @note When combining adaptively refined meshes with hanging node
    * constraints and boundary conditions like from the current function within
    * one AffineConstraints object, the hanging node constraints should always
-   * be set first, and then the boundary conditions since boundary conditions
+   * be set first and then the boundary conditions, since boundary conditions
    * are not set in the second operation on degrees of freedom that are
    * already constrained. This makes sure that the discretization remains
    * conforming as is needed. See the discussion on conflicting constraints in
-   * the module on
-   * @ref constraints.
+   * the module on @ref constraints.
    *
-   * This function is fundamentally equivalent to the ones above except that it
-   * puts its results into an AffineConstraint object rather than a `std::map`.
-   * See the functions above for more comments.
+   * TODO how to refer to interpolate_boundary_values() function with most
+   * detailed documentation (it appears on a different page in Doxygen)?
```

Comment:
Unfortunately, I don't know a solution here. I would refer to this with a general text
```suggestion
   * For further information and details on the other function arguments, see the
   * interpolate_boundary_values() function with `std::map` arguments and the
   * general class documentation.
```
Of course, if someone knows a better way that creates a doxygen link, that would be even better.

Reply:
I think this is the way to go.